src/components/Compare.vue

```diff
@@ -1,37 +1,91 @@
 <template>
   <div>
-    <div v-for="trace in Object.keys(traces)" :key="trace">
-      <input type="checkbox" :id="trace" :value="trace" v-model="selected" />{{ trace }}
-    </div>
+
     <!-- {{ csvs }} -->
     <remote-component
       url="https://unpkg.com/vue-plotly@^1/dist/vue-plotly.umd.min.js"
-      :data="[{ x: [0, 1, 2, 3], y: [1, 3, 3, selected.length], type: 'scatter' }]"
+      :data="csvs"
+      :layout="layout"
       :extract="library => library.Plotly" />
+      <div v-for="trace in Object.keys(traces)" :key="trace">
+      <input type="checkbox" :id="trace" :value="trace" v-model="selected" />{{ trace }}
+    </div>
   </div>
 </template>
 <script lang="ts">
 import { Component, Prop, Vue } from "vue-property-decorator";
-import axios from "axios";
 import AsyncComputed from "vue-async-computed-decorator";
 import RemoteComponent from "vue-remote-component";
+import { parse, ParseResult } from 'papaparse';
 
 @Component({
   components: { RemoteComponent },
 })
 export default class Compare extends Vue {
   @Prop({required: true, type: Object}) traces!: { [key: string]: string };
 
+  layout = {
+      xaxis: {
+          title: {
+              text: '<i>E</i> [V]',
+              font: {
+                  family: 'Courier New, monospace',
+                  size: 18,
+              }
+          },
+          showline: true,
+          zeroline: false,
+          ticks: "outside",
+          mirror: true,
+          side: "bottom"
+      },
+      yaxis: {
+          title: {
+              text: '<i>j</i> [A m⁻²]',
+              font: {
+                  family: 'Courier New, monospace',
+                  size: 18,
+              }
+          },
+          showline: true,
+          zeroline: true,
+          ticks: "outside",
+          mirror: true,
+          side: "left"
+      },
+      legend: {
+          yanchor: "top",
+          y: -0.3,
+          xanchor: "left",
+          x: 0.01
+      }
+  };
+
   selected: string[] = [];
 
   @AsyncComputed({default: []})
   async csvs() {
```

Comment:
This is not async anymore and the asynchronous bits here just happens to work but won't work reliably.

Instead you need to create a proper promise here that you can await:
```
{
const parsed = await new Promise((complete, error) => parse(..., {..., complete, error}));
build x and y
csvs.push()
}
```
```